packages/web-components/fast-foundation/src/checkbox/README.md

```diff
@@ -7,31 +7,74 @@ custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-comp
 
 An implementation of a [checkbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox) as a form-connected web-component.
```

Comment:
Should we call out that this is an ARIA implementation? There are different requirements, such as the `<label>` element not being applicable to ARIA checkboxes (and Radios)

Reply:
@chrisdholt Can you paste me the content that you want to put here instead? Or submit an insert suggestion that I can easily merge? You know better than I what the wording should be.